### PR TITLE
Handle SwissProt UnknownPosition with null BioSQL location

### LINKED ISSUE #1165: Handling UnknownPosition
I'm working on a BioSQL database for SwissProt entries and it's balking at some of the UnknownPosition values that are being set by the SeqIO parser (I'm using BioPython 1.69).  I'm sure wading into the fuzzy location morass is not a fun prospect, but I'm wondering if this has been given some thought.  I'm happy to try to implement any reasonable suggestion. 

For reference, the specific error that comes up is in BioSQL/Loader.py line 912.  Uniprot record P97881 is an example:
    
      File "/n/home_rc/akitzmiller/.conda/envs/2.7.13/lib/python2.7/site-packages/biopython-1.69-py2.7-linux-x86_64.egg/BioSQL/Loader.py",  line 912, in _insert_location
          end = int(location.end)
      TypeError: int() argument must be a string or a number, not 'UnknownPosition'

### PR DESCRIPTION
**Pull request by aaronk**
fixes #1165
If a feature location element is of type UnknownPosition, it is stored as a null value in the BioSQL location table.  BioSQL start or end columns with a null value are retrieved as UnknownPosition.  This is mainly an issue with SwissProt records.

### CODE REVIEW

File: Doc/examples/tree1.nwk [hunk 1]
```diff
@@ -0,0 +1 @@
+((A:0.10200,B:0.23000):0.06000,C:0.40000):0.00000;
```
Comment (peterjc) [inline Doc/examples/tree1.nwk]: This file looks to have been included by mistake?

**Comment by peterjc** [comment]
Rather than adding an extra XML test file, would existing file ``Tests/SwissProt/multi_ex.xml`` serve here?

**Comment by aaronk** [comment]
Sure.

ajk

**Comment by aaronk** [comment]
Not sure what that is.  I can get rid of it.

ajk

**Comment by peterjc** [comment]
Cross reference #1165, where I noted that the pydocstyle issue should be fixed on the master branch. The test failure ought to go away after you update the branch (e.g. removing that stray file).

**Comment by aaronk** [comment]
Got rid of the nwk file (seems to be generated when running the test suite).  I'm going to leave the extra Swissprot file in place- adding the entry to multi_ex.xml opened a can of worms.

ajk

**Comment by peterjc** [comment]
OK then :)

Are you happy to dual licence your contributions under both the _Biopython License Agreement_ AND the _BSD 3-Clause License_? See #898.

Also, are you happy to be thanked by name in the ``NEWS.rst`` and ``CONTRIB.rst`` files?

**Comment by aaronk** [comment]
Yes on both accounts.  

ajk

**Comment by peterjc** [comment]
Thank you

**Pull request merged**